applications/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -20,9 +21,28 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     );
   }
 
-  const portfolioId = event.pathParameters?.portfolioDraftId;
-
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+  if (!IsValidJson(event.body)) {
```

Comment:
`IsValidJson` doesn't match proper naming conventions. I'll try to remember to mention this again at it's definition